source/manual/databases.html.md

```diff
@@ -0,0 +1,37 @@
+---
+owner_slack: "#govuk-2ndline-tech"
+title: Databases on GOV.UK
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+---
+
+Many GOV.UK applications use a database. See our [spreadsheet of applications and database engines](https://docs.google.com/spreadsheets/d/1rBZeeaT9XevaRvstMjw2YsbSsoB7IqgWjpJz5oAO9nM/edit#gid=1368371571) for details.
+
+## Database engines
+
+GOV.UK uses mostly PostgreSQL and some MySQL for apps that require a relational database. There's no particular reason why we use both, and the fact that we do is considered [tech debt](https://trello.com/c/zlfgSJlV/69-govuk-uses-mysql-and-postgresql-and-mongo).
+
+GOV.UK uses MongoDB for apps that require a document database. MongoDB is synonymous with DocumentDB; the name describes where how it is hosted, but the underlying engine is the same.
+
+Finally, GOV.UK uses an ElasticSearch database for search.
+
+## Hosting
+
+Instead of running these databases locally on the same instance as the application, they're hosted in separate infrastructure.
```

Comment:
With two exceptions, outlined here: https://docs.publishing.service.gov.uk/repos/govuk-aws/guides/rds-database-management.html.

Not sure if that's worth calling out in this doc or not.

Reply:
It might be worth including your first sentence, I reckon. They're both going to hopefully go away soon, but you know how things go 😄